src/Tribe/Attendees.php

```diff
@@ -661,9 +661,18 @@ public function maybe_generate_csv() {
 			// Create the file pointer connected to the output stream.
 			$output = fopen( 'php://output', 'w' );
 
+			/**
+			 * Allow for modifying the field delimiter in the CSV export file.
+			 *
+			 * @since 5.1.2
```

Comment:
```suggestion
			 * @since TBD
```

Even though the logical thing is that the number will be 5.1.2 we should leave it as TBD so it's replaced when packaging (That's the norm in case we change version name or we have a hotfix or something).